dm/tests/duplicate_event/run.sh

```diff
@@ -73,11 +73,22 @@ function run_with_prepared_source_config() {
 	run_sql_source2 "show master status;"
 	binlog_file=$(grep "File" $TEST_DIR/sql_res.$TEST_NAME.txt | awk -F: '{print $2}' | xargs)
 	binlog_pos=$(grep "Position" $TEST_DIR/sql_res.$TEST_NAME.txt | awk -F: '{print $2}' | xargs)
-
 	server_uuid=$(tail -n 1 $WORK_DIR/worker2/relay-dir/server-uuid.index)
-	relay_log_size=$(ls -al $WORK_DIR/worker2/relay-dir/$server_uuid/$binlog_file | awk '{print $5}')
-	echo "binlog_pos: $binlog_pos relay_log_size: $relay_log_size"
-	[ "$binlog_pos" -eq "$relay_log_size" ]
+
+	succ=0
```

Comment:
Seems that we just retry this case and make it run more times. Do we know the root cause of the relay log being too slow? Do we have another approach to speed up the relay log?

Reply:
retry is expected. As in PR description, for the sequence [..., RowsEvent, XIDEvent], the test script will reach here after  relay writes RowsEvent and syncer processed RowsEvent. But relay may have no time to write XIDEvent.